browser/static/js/main_track.js

```diff
@@ -545,72 +544,70 @@ function label_under_mouse() {
 }
 
 function render_highlight_info() {
+  const highlightText = (current_highlight) ? 'ON' : 'OFF';
+  let currentlyHighlighted = 'none';
   if (current_highlight) {
-    $('#highlight').html("ON");
-    if (mode.highlighted_cell_one !== -1) {
-      if (mode.highlighted_cell_two !== -1) {
-        $('#currently_highlighted').html(mode.highlighted_cell_one + " , " + mode.highlighted_cell_two);
-      } else {
-        $('#currently_highlighted').html(mode.highlighted_cell_one);
-      }
+    if (edit_mode) {
+      currentlyHighlighted = (brush.value > 0) ? brush.value : '-';
     } else {
-      $('#currently_highlighted').html("none");
+      if (mode.highlighted_cell_one !== -1) {
+        if (mode.highlighted_cell_two !== -1) {
+          currentlyHighlighted = `${mode.highlighted_cell_one}, ${mode.highlighted_cell_two}`;
+        } else {
+          currentlyHighlighted = mode.highlighted_cell_one;
+        }
+      }
     }
-  } else {
-    $('#highlight').html("OFF");
-    $('#currently_highlighted').html("none");
   }
+  document.getElementById('highlight').innerHTML = highlightText;
+  document.getElementById('currently_highlighted').innerHTML = currentlyHighlighted;
 }
 
 function render_edit_info() {
-  if (edit_mode) {
-    $('#edit_mode').html('pixels');
-    $('#edit_brush_row').css('visibility', 'visible');
-    $('#edit_label_row').css('visibility', 'visible');
-    $('#edit_erase_row').css('visibility', 'visible');
+  const editModeText = (edit_mode) ? 'paint mode' : 'whole-label mode';
+  document.getElementById('edit_mode').innerHTML = editModeText;
 
-    $('#edit_brush').html(brush.size);
-    $('#edit_label').html(brush.value);
+  const rowVisibility = (edit_mode) ? 'visible' : 'hidden';
+  document.getElementById('edit_brush_row').style.visibility = rowVisibility;
+  document.getElementById('brush_label_row').style.visibility = rowVisibility;
+  document.getElementById('edit_erase_row').style.visibility = rowVisibility;
 
-    if (brush.erase) {
-      $('#edit_erase').html("ON");
-    } else {
-      $('#edit_erase').html("OFF");
-    }
+  if (edit_mode) {
+    document.getElementById('edit_brush').innerHTML = brush.size;
 
-  } else {
-    $('#edit_mode').html('whole labels');
-    $('#edit_brush_row').css('visibility', 'hidden');
-    $('#edit_label_row').css('visibility', 'hidden');
-    $('#edit_erase_row').css('visibility', 'hidden');
+    const editLabelText = (brush.value > 0) ? brush.value : '-';
+    document.getElementById('brush_label').innerHTML = editLabelText;
+
+    const editEraseText = (brush.erase && !brush.conv) ? 'ON' : 'OFF';
+    document.getElementById('edit_erase').innerHTML = editEraseText;
```

Comment:
Doesn't seem to be a difference for our use. innerHTML would allow us to include HTML tags if we wanted to format the text, but we're not doing that

Reference: https://medium.com/better-programming/whats-best-innertext-vs-innerhtml-vs-textcontent-903ebc43a3fc